bundles/net.tourbook.export/src/net/tourbook/export/ExportTourFIT.java

```diff
@@ -30,10 +31,14 @@ public class ExportTourFIT extends ExportTourExtension {
    /**
     * Plugin extension constructor
     */
-   public ExportTourFIT() {}
+   public ExportTourFIT() {
+
+      setImageDescriptor(Activator.getImageDescriptor(
+            UI.IS_DARK_THEME ? ExportImages.Export_FIT_Logo_Dark : ExportImages.Export_FIT_Logo));
+   }
```

Comment:
I would have written this code with this code formatting

![image](https://user-images.githubusercontent.com/1283445/236380664-c91150d5-03dd-4d80-bcc5-17f218dee2a4.png)

Reply:
Ok, I will fix

Reply:
Fixed